_data/simple-icons.json

```diff
@@ -7650,7 +7655,7 @@
         {
             "title": "ResearchGate",
             "hex": "00CCBB",
-            "source": "https://c5.rgstatic.net/m/428059296771819/images/favicon/favicon.svg"
+            "source": "https://apps.apple.com/us/app/researchgate/id1208163937?ls=1"
```

Comment:
I'm not sure for this one since there are no vector version in this page.

Reply:
the file is a webp file. That's as good as an SVG. It can be exported to SVG with Sketch app without any reconfiguration.

Reply:
A raster file format is never _**as**_ good as a vector file. Even if it is easy to import/export, the quality is rarely the same. I'm still not sure on this one.

Reply:
It's not a problem for every logo, however. This is a square with rounded corners and two letters in it. It's not like the eye in Cassandra or the feather in JMeter.
The problematic of having hi res sources is more or less important depending on the complexity of what we are trying to achieve.
If it were a face, or a fingerprint, perhaps it would be best to have a hi res SVG as source. BUt for things like Gmail icons or TuneIn or ResearchGate, it doesn't really matter all that much: it's all letters and lines.

Reply:
If a SVG was used as source, it is wrong to change the source for a SVG as simple the icon looks.

Just bringing back the previous comment from @ericcornelissen which, kind of, disappeared 😕 

> The source should probably just be https://www.researchgate.net/, as the favicon that is the current source is used there.
>
> @jorgeamadosoria, to make it perhaps a bit easier to understand the source choice, could you include an explanation in the commit making this change (or do it in a dedicated PR for visibility)?

Reply:
I already set the source to the homepage.